imap_processing/quality_flags.py

```diff
@@ -0,0 +1,63 @@
+"""Bitwise flagging."""
+
+from enum import IntFlag
+
+
+class L1bQualityFlags(IntFlag):
+    """Quality flags."""
+
+    NONE = 0x0
+
+    BASE_INF = 0x1  # bit 0, Infinite value
```

Comment:
I don't think that `BASE_` or `ENA_` should be included in the names. More flexibility is afforded by not assuming any value/name overlap across data levels or products.